packages/core/package.json

```diff
@@ -63,6 +63,7 @@
     "@upstash/redis": "^1.34.3",
     "ai": "^4.0.3",
     "anthropic-vertex-ai": "^1.0.2",
+    "composio-core": "^0.3.2",
     "date-fns": "^3.0.5",
     "express": "^4.21.1",
     "inngest": "3.22.3",
```

Comment:
Random question as I am just seeing this. Do we still need `inngest` as a dependency in core?

Reply:
nope